client/components/site-selector/index.jsx

```diff
@@ -185,7 +185,7 @@ const SiteSelector = React.createClass( {
 					href={ this.props.siteBasePath ? siteHref : null }
 					key={ 'site-' + site.ID }
 					indicator={ this.props.indicator }
-					onSelect={ this.onSiteSelect.bind( this, site.slug ) }
+					onSelect={ this.onSiteSelect.bind( this, site.slug, site.ID ) }
```

Comment:
Well, I think that if we touch a line which could be improved in a PR, we should improve it.